webrtc.html

```diff
@@ -2903,6 +2903,12 @@
                     "RTCIceTransportState">closed</a></code>.</p>
                   </li>
                   <li>
+                    <p>Set the <var>connection</var>'s
+                    <code>sctp.transport.state</code>
+                    to <code><a data-link-for=
+                    "RTCDtlsTransportState">closed</a></code>.</p>
+                  </li>
+                  <li>
```

Comment:
We could define a "collect set of `RTCDtlsTransport`s" algorithm, as we did for "set of `RTCRtpSender`s". Though it looks like that was removed at some point? Anyway, if we did this, then we could just have "for each transport in the set of transports", and would avoid this algorithm touching the same transport multiple times.